src/components/FormContainer/FormContainer.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Component, createContext } from 'react';
+
+export const FormContext = createContext(null);
```

Comment:
rewrite on functional component with hook's

Reply:
left a todo in code